### PR TITLE
Update timeout in the read replica haproxy configuration

### PR DESCRIPTION
**Pull request by frankhereford**
## Associated issues

This PR aims to close both https://github.com/cityofaustin/atd-data-tech/issues/14987 and https://github.com/cityofaustin/atd-data-tech/issues/16226.

## Clarification :reminder_ribbon: 

I had stated that I had not yet pushed up and merged this configuration, but I had. I just didn't remember doing it. It's been in the toolbox all along, and this PR is an update to that configuration.

#### This is deployed in production already.

* [ ] If you can use the read replica for VZ, then this is working. 
* [ ] Additionally, you can test to see if you don't have your connection dropped until [10 minutes of being idle](https://github.com/cityofaustin/atd-vz-data/commit/f75648d9e16ee0fd090b198e14f13b5487894419).

---
#### Ship list

- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-toolbox/read_replica_proxy/haproxy.cfg.template [hunk 1]
```diff
@@ -5,8 +5,8 @@ defaults
   log global
   mode tcp
   timeout connect 5000ms
-  timeout client 50000ms
-  timeout server 50000ms
+  timeout client 600000ms
```
Comment (mddilley) [inline atd-toolbox/read_replica_proxy/haproxy.cfg.template:8]: i think that these can be [represented in minutes](https://www.haproxy.com/documentation/haproxy-configuration-manual/latest/#2.5) if we wanted. Not really a big deal - i was just curious about it myself.

**Review by mddilley — APPROVED** [review]
Looks great! I found [this StackExchange thread](https://serverfault.com/questions/504308/by-what-criteria-do-you-tune-timeouts-in-ha-proxy-config) that helped me understand the config more if anyone is interested. 10 minutes sounds good because I saw some posts that suggest starting at 5. 🚢 🙌

File: atd-toolbox/read_replica_proxy/haproxy.cfg.template (hunk 1 above)
Comment (frankhereford) [reply atd-toolbox/read_replica_proxy/haproxy.cfg.template]: Such a good call! Thank you - this is much more readable. https://github.com/cityofaustin/atd-vz-data/pull/1397/commits/a801f9a1b09dcbebcae3a32c858ce351a71c0f89

**Review by chiaberry — APPROVED** [review]

**Review by roseeichelmann — APPROVED** [review]
thanks frank! this would happen to me sometimes but it just never bothered me too much, nice to have it fixed though!

**Review by johnclary — APPROVED** [review]
thanks! 🚢

**Pull request merged**